CHANGELOG.md

```diff
@@ -1,3 +1,154 @@
+# 82.0.0
+
+In this major release, we dropped support for iOS 8 in all components. We also introduced new method and properties for AppBar, BottomSheet and NavigationDrawer.
+
+## Breaking changes
+
+iOS 8 support has been dropped in all components. This change will require application bumping minimum depolyment target to iOS 9.
+
+## New features
+
+### Add support for MDCAppBarNavigationController delegate to pick a tracking scroll view
+
+We added a method in delegate for clients to be able to customize the tracking scroll view detection logic. 
+
+#### Swift
+
+```swift
+func appBarNavigationController(_ navigationController: MDCAppBarNavigationController,
+                              trackingScrollViewFor trackingScrollViewForViewController: UIViewController,
+                              suggestedTrackingScrollView: UIScrollView?) -> UIScrollView? {
+  return trackingScrollView
+}
+```
+
+#### Objective-C
+
+```objc
+- (UIScrollView *)appBarNavigationController:(MDCAppBarNavigationController *)navigationController
+                  trackingScrollViewForViewController:(UIViewController *)viewController
+                          suggestedTrackingScrollView:(UIScrollView *)scrollView {
+  return self.trackingScrollView;
+}
+```
+
+### `flashScrollIndicators` in BottomSheet
+
+We added a behavioral flag for when clients want to flash scroll indicators so that clients can opt into this behavior.
+
+#### Swift
+
+```swift
+bottomSheet.shouldFlashScrollIndicatorsOnAppearance = true;
+
+```
+
+#### Objective-C
+
+```objc
+self.bottomSheet.shouldFlashScrollIndicatorsOnAppearance = YES;
+
+```
+
+### `maximumInitialDrawerHeight` in NavigationDrawer
+
+We added this new property for NavigationDrawer's API, which allows clients to set the initial height of the drawer, rather than it always being 50% of the screen's height.
```

Comment:
Same tip about personal pronouns.  Suggestion (subject to accuracy):

"Clients can now override the initial drawer height on `MDCBottomDrawerViewController` with a desired height in pixels.  This value will be used if it is greater than the `preferredContentSize` of the Bottom Drawer."

Reply:
> This value will be used if it is greater than the preferredContentSize of the Bottom Drawer.

Shouldn't it be "the value will be used if it is less than the `preferredContentSize`?

Reply:
@yarneo to confirm.

Reply:
Addressed in 9f0231e5fdba3e562031a4cfcc6c565c54a1e08e